readme.md

```diff
@@ -130,6 +130,7 @@ Here you find projects that were made especially for hacktoberfest.
 - [Grafana Labs](https://github.com/grafana/grafana)
 - [Hasura](https://blog.hasura.io/hasura-joins-hacktoberfest-2019/)
 - [JetBrains](https://www.jetbrains.com/lp/hacktoberfest-2020/)
+```JetBrains offers a wide range of developer IDEs. For Hacktoberfest, it makes them available for free for open source contributors independently of the contributor being a permanent contributor to an open source project (which is covered by their Free Licenses Program)```
```

Comment:
This is not the format for an awesome-list. Are you aware of some other awesome lists such as https://github.com/sindresorhus/awesome#contents?